cmd/docs.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/TouchBistro/goutils/fatal"
+	"github.com/TouchBistro/tb/config"
+	"github.com/spf13/cobra"
+)
+
+var docsCmd = &cobra.Command{
+	Use:   "docs",
+	Args:  cobra.MinimumNArgs(1),
+	Short: "Opens link to API docs for a given service",
```

Comment:
Can you add a `Long` with a detailed description of how to use it? You can scope some of the other commands for examples